apps/nfw-react/src/app/project/cmp/Cart/_cart.scss

```diff
@@ -0,0 +1,129 @@
+.cart {
+  display: grid;
+  grid-template-columns: 30px auto 30px;
+  grid-template-rows: 10vh 6rem 60vh;
+
+  grid-template-areas:
+    '. . .'
+    '. cart_heading .'
+    '. cart_container .';
+
+  &__heading {
+    grid-area: cart_heading;
+    justify-self: center;
+    align-self: center;
+    font-size: 2.5rem;
+    margin-bottom: 0.5rem;
+    color: $secondary-color;
+    transition: 0.5s ease-out;
+
+    &--one {
+      font-size: 1.8rem;
+    }
+
+    &--two {
+      font-size: 1.3rem;
+
+      &:hover {
+        color: $tertiary-color;
+      }
+    }
+  }
+
+  &__container {
+    grid-area: cart_container;
+    justify-self: stretch;
+    min-width: 260px;
+    display: grid;
+    grid-template-rows: 75% 25%;
+    grid-template-columns: 100%;
+    justify-content: center;
+    padding: 0.5rem 1rem 1rem;
+    border-radius: 15px;
+
+    background-color: $primary-color;
+
+    &--empty {
+      grid-template-rows: 90% 10%;
+    }
+  }
+
+  &__list {
+    width: 100%;
+    list-style-type: none;
+    justify-self: stretch;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__item {
+    margin: 7px;
+    text-align: center;
+    font-size: 1.3rem;
+    color: $secondary-color;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+
+    &:hover {
+      text-decoration: line-through;
+    }
+
+    .extra-ingredients {
```

Comment:
(should) extra-ingredients block doesn't seems to be connected with previous block so it can be outside of the .cart class (unnecessary complication + will be less readable) -> It's good point to think about new component for that part